static/js/theme-map/Renderer/ElementRenderTarget.js

```diff
@@ -47,11 +47,11 @@ class ElementRenderTarget extends RenderTarget {
 
   /**
    * async render(data) => DOMNode
-   * Calls templateFunction(data) and sets value to element.innerHTML, then returns element
+   * Calls templateFunction(data) and sets value to element.textContnet, then returns element
```

Comment:
Seems like we have a typo here? I imagine it's supposed to be textContent right?

Reply:
Nice catch! Updated

Reply:
looks like the type in this doc is still not updated

Reply:
lol, updated